<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryFilterDialog</class>
 <widget class="QDialog" name="EntryFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start Practice</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;p&gt;Your selection of vocabulary for the practice is empty.&lt;/p&gt;&lt;p&gt;Below you can choose to ignore some of your configuration settings to start a practice anyway.&lt;/p&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ignoreGroupBox">
     <property name="title">
      <string>Ignore Configuration for:</string>
     </property>
     <layout class="QGridLayout" name="_2">
      <item row="1" column="0">
       <widget class="QCheckBox" name="lessonCheckBox">
        <property name="text">
         <string>Ignore Lessons</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lessonLabel">
        <property name="text">
         <string comment="KDE::DoNotExtract">TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="wordTypeCheckBox">
        <property name="text">
         <string>Ignore Word Types</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="wordTypeLabel">
        <property name="text">
         <string comment="KDE::DoNotExtract">TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="blockedCheckBox">
        <property name="text">
         <string>Ignore Blocked</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="blockedLabel">
        <property name="text">
         <string comment="KDE::DoNotExtract">TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="timesWrongCheckBox">
        <property name="text">
         <string>Ignore Times Answered Incorrectly</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="timesWrongLabel">
        <property name="text">
         <string comment="KDE::DoNotExtract">TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="timesPracticedCheckBox">
        <property name="text">
         <string>Ignore Times Practiced</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="timesPracticedLabel">
        <property name="text">
         <string comment="KDE::DoNotExtract">TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="minMaxGradeCheckBox">
        <property name="text">
         <string>Ignore Minimum/Maximum Grade</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="minMaxGradeLabel">
        <property name="text">
         <string comment="KDE::DoNotExtract">TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="numberAffectedWordsLabel">
        <property name="text">
         <string>Number of Affected Words</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="documentTotalLayout">
     <item>
      <widget class="QLabel" name="documentTotalDescriptionLabel">
       <property name="text">
        <string>Total number of entries:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="documentTotalLabel">
       <property name="text">
        <string comment="KDE::DoNotExtract">TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="totalLayout">
     <item>
      <widget class="QLabel" name="totalDescriptionLabel">
       <property name="text">
        <string>Resulting number of words to practice:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="totalLabel">
       <property name="text">
        <string comment="KDE::DoNotExtract">TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>57</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
